packages/vite-plugin-cloudflare/playground/module-resolution/__tests__/module-resolution.spec.ts

```diff
@@ -1,11 +1,11 @@
+// Note: this file should be kept in sync with ./nodejs-compat/module-resolution.spec.ts
```

Comment:
this is annoying... but I don't think there's a way to do this better 😓 

If we were to copy the playground in a playground-tmp folder like we used to probably we could come up with some convention that could copy files over multiple variants, I can't remember why we removed the tmp folder, but I guess it's not worth reintroducing it for this

any thoughts? 🤔

Reply:
If you think it is worth it, I'm not against running these tests in tmp directories (at least in CI), as long as it is still easy enough to cd into a playground and get it up and running locally without jumping through too many hoops.

Reply:
I was just mentioning it, I am not really sure if it is worth it... do you remember why we removed the tmp directory? was it causing issues?

If we don't re-introduce a tmp directory, are you ok with this testing code duplication? and/or have any ideas on how to avoid this?

Reply:
> To be honest adding complexity where we copy certain files between directories when creating the tmp projects for testing seems to be quite magical and not that obvious. So I think that a manual sync is acceptable.
> 
> What about using a symlink between the files?

I did consider symlinks but I don't think that vitest would cope with those, even if it did I am worried about other OSes (that we test in CI), are symlinks something that other OSes can understand? (sorry if it's a silly question)

> Or just importing the test cases between playgrounds?

I did though of that too thinking that it wouldn't work, but let me actually give it a quick try 👍

Reply:
oh! 😮 

Surpisingly (to me at least 😅) this actually works: https://github.com/cloudflare/workers-sdk/pull/8489/commits/0cef2999d85143231193135eaff16897b738b4ca

This is awesome! I'm really happy with this solution, WDTY? 🙂

Reply:
Marvellous!